### DISCUSSION TITLE
About plans

### DISCUSSION THREAD

**Discussion by me-nkr** [daattali/beautiful-jekyll#988] [Q&A] [answered]
Just wondering, if I can fork the repo and change the code, what is the point of the plans?

**Comment by daattali** [comment, marked as answer]
The lowest plan is just to show support/appreciation. The other plans are for getting specific extra benefits. If you don't need any of the benefits, you don't have to purchase a plan. You're free to fork and do whatever you want with the code (although a credit to the original author is generally regarded as good practice)

**Reply by me-nkr** [reply]
The feature I find interesting is the remove ads one, except I still haven't figured out what the ad is : \
Also if I fork it will the ads be there too?

**Reply by daattali** [reply]
By ad I mean the phrase at the bottom "Powered by beautiful jekyll"
